### PR TITLE
Allow setting TCP and UDP service annotations separately

### PR DESCRIPTION
**Pull request by nicolaschan**
### What does this PR do?

Adds the following two options under `service`:
- `annotationsTCP: {}`
- `annotationsUDP: {}`

These annotations are applied to either the TCP or UDP service *only*. The option `service.annotations` remains and keeps the existing behavior of applying annotations to both services.

### Motivation

I am using [kubernetes-sigs/external-dns](https://github.com/kubernetes-sigs/external-dns) to set DNS records for my traefik services. DNS records are specified by annotating the service. Traefik creates TCP and UDP LoadBalancer services and each is assigned a different IP address by [metallb](https://metallb.universe.tf/). I would like to set DNS records independently for the TCP service and UDP service, but this is not currently possible because `service.annotations` applies to *both* services.

There are likely other cases where you would want different annotations between the TCP and UDP services.

### More

- [x] Yes, I updated the [chart version](https://github.com/traefik/traefik-helm-chart/blob/9b32ed1b414cc0be1ad46bcb335fcfc93ded06f3/traefik/Chart.yaml#L5)

### Additional Notes

Open to design suggestions. I considered doing something like:
```yaml
service:
  annotations:
    tcp: {} # TCP-only annotations here
    udp: {} # UDP-only annotations here
```
This would break existing behavior. I think it is better to keep existing values working, so that is why I added `annotationsTCP` and `annotationsUDP` where I did.

### CODE REVIEW

File: traefik/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: traefik
 description: A Traefik based Kubernetes ingress controller
 type: application
-version: 10.1.2
+version: 10.2.0
```
Comment (jakubhajek) [inline traefik/Chart.yaml:5]:
```suggestion
version:  10.1.7
```

**Review by jakubhajek — COMMENTED** [review]
Thank you a lot for contributing to Traefik Helm Chart. I played a little bit with your PR and haven't experienced any issues, so we should be fine with merging that. Can you please just update the version of the chart?

**Review by jakubhajek — APPROVED** [review]
LGTM

File: traefik/Chart.yaml (hunk 1 above)
Comment (nicolaschan) [reply traefik/Chart.yaml]: Chart version updated

**Comment by nicolaschan** [comment]
@jakubhajek Sorry, looks like there was a merge conflict on the version. I've fixed it, so might need re-approval.

**Review by jakubhajek — APPROVED** [review]
Ready to be merged ;)

File: traefik/Chart.yaml (hunk 1 above)
Comment (jakubhajek) [reply traefik/Chart.yaml]: My mistake. We are following semver, then: 

- enhancement: 1.X.0
- bug: 1.1.X
- breaking: x.0.0

The version you initially set was correct as it is an enhancement.

**Review by jakubhajek — APPROVED** [review]
The chart version is set to 10.2.0 as we are adding a new enhancement. Looks good to me.

**Pull request merged**